.github/workflows/build-system-test.yml

```diff
@@ -0,0 +1,145 @@
+# Description: this workflow runs build mega-apps that have all the connected components.
+
+name: Build System Test Canary
+
+on:
+  schedule:
+    - cron: '*/15 * * * *' # Run every 15 minutes
+  push:
+    branches: [main] # Run when merge to main
```

Comment:
Should we run this on `push`? I don't see much value in this, because `build-system-tests` will test against `@latest`, not the commit that was just merged.

Reply:
Oh, that might be my mis-understanding. I thought `push` on `main` is publish to `@latest` 😂

Reply:
`push` on `main` is by default publish to `@next` unless the commit is from "Version Packages" PR. In either way, we would have to wait until `publish-next` finishes to run build test against it.

Reply:
Done b645eeb9652c22639a7836b1941d968322b98c8d